File: phyton/Petshop.py
class Petshop:
    # konstruktor
    def __init__(self, id, name, category, price):
        self.id = id
        self.name = name
        self.category = category
        self.price = price
        
    # getter dan setter untuk id
    def setId(self, id):
        self.id = id
    
    def getId(self):
        return self.id
    
    # getter dan setter untuk name
    def setName(self, name):
        self.name = name
    
    def getName(self):
        return self.name
    
    # getter dan setter untuk category
    def setCategory(self, category):
        self.category = category
    
    def getCategory(self):
        return self.category
    
    # getter dan setter untuk price
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price

File: php/Petshop.php
<?php
class Petshop
{
    // atribut private
    private $id;
    private $name;
    private $category;
    private $price;
    private $image;

    // konstruktor dengan parameter untuk mengisi nilai
    public function __construct($id, $name, $category, $price, $image)
    {
        $this->id = $id;
        $this->name = $name;
        $this->category = $category;
        $this->price = $price;
        $this->image = $image;
    }

    // getter dan setter untuk id
    public function setId($id)
    {
        $this->id = $id;
    }

    public function getId()
    {
        return $this->id;
    }

    // getter dan setter untuk name
    public function setName($name)
    {
        $this->name = $name;
    }

    public function getName()
    {
        return $this->name;
    }

    // getter dan setter untuk category
    public function setCategory($category)
    {
        $this->category = $category;
    }

    public function getCategory()
    {
        return $this->category;
    }

    // getter dan setter untuk price
    public function setPrice($price)
    {
        $this->price = $price;
    }
    
    public function getPrice()
    {
        return $this->price;
    }

    // getter dan setter untuk image
    public function setImage($image)
    {
        $this->image = $image;
    }

    public function getImage()
    {
        return $this->image;
    }

    function __destruct() {}
}
